src/PKSim.Core/Commands/EditParameterCommand.cs

```diff
@@ -9,9 +9,10 @@ namespace PKSim.Core.Commands
    public abstract class EditParameterCommand : BuildingBlockChangeCommand
    {
       protected IParameter _parameter;
-      protected ValueOrigin _oldValueOrigin;
```

Comment:
This is now gone from the command and is being dealt with from another command. Separation of concerns